tests/test_mesh.py

```diff
@@ -201,4 +201,30 @@ def test_3d_single_tetrahedron(self):
                           nbound=4, ngstnode=4, ngstface=12, ngstcell=4,
                           nedge=6)
 
+    def test_1d_single_line(self):
+        mh = modmesh.StaticMesh(ndim=1, nnode=2, nface=0, ncell=1)
+        mh.ndcrd.ndarray[:] = [[0], [1]]
+        mh.cltpn.ndarray[:] = modmesh.StaticMesh.LINE
+        mh.clnds.ndarray[:, :3] = [(2, 0, 1)]
+
+        self._check_shape(mh, ndim=1, nnode=2, nface=0, ncell=1,
+                          nbound=0, ngstnode=0, ngstface=0, ngstcell=0,
+                          nedge=0)
+        self._check_metric_trivial(mh)
+
+        mh.build_interior(_do_metric=False, _build_edge=False)
+        self._check_shape(mh, ndim=1, nnode=2, nface=2, ncell=1,
+                          nbound=0, ngstnode=0, ngstface=0, ngstcell=0,
+                          nedge=0)
+        self._check_metric_trivial(mh)
+
+        mh.build_interior()  # _do_metric=True, _build_edge=True
+        self._check_shape(mh, ndim=1, nnode=2, nface=2, ncell=1,
+                          nbound=0, ngstnode=0, ngstface=0, ngstcell=0,
+                          nedge=2)
+
+        # _do_metric do nothing due to dim == 1
+        self._check_metric_trivial(mh)
+        # TODO: Need to add build_boundary and build_ghost to make sure
```

Comment:
I left a comment to mention that there are some test need to be added in the future.

Reply:
I see.  Thanks.